common/wbdisable/wbdisable-fr.html

```diff
@@ -0,0 +1,22 @@
+---
+{
+	"title": "GCWeb Disable (wb-disable) example",
+	"language": "fr",
+	"altLangPage": "wb-disable-en.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-fr.html" }
+	],
+	"dateModified": "2023-07-04"
+}
+---
+<div class="wb-prettify all-pre"></div>
+<p>Cette page nécessite une traduction.</p>
+<div lang="en">
+	<h2>Basic HTML mode</h2>
+	<p>Click on the link below to enable Basic HTML Mode</p>
+	<a href="?wbdisable=true">Switch to Basic HTML Mode</a>
```

Comment:
Here the translated content

```suggestion
<p>Cliquez sur le lien ci-dessous pour activer le mode HTML simplifié</p>
<p><a href="?wbdisable=true">Basculer vers le mode HTML simplifié</a></p>
<p><a href="?wbdisable=false">Basculer vers la page régulière</a></p>
```